src/3d/qgscamerapose.cpp

```diff
@@ -62,8 +62,11 @@ void QgsCameraPose::setPitchAngle( float pitch )
   // prevent going over the head
   if ( pitch > 180.0f )
     mPitchAngle = 180.0f;
-  else if ( pitch < 0.0f )
-    mPitchAngle = 0.0f;
+  else if ( pitch < 0.2f )
```

Comment:
```suggestion
  else if ( pitch < 0.2f )
```

Maybe use `std::clamp` from algorithm include:

`std::clamp(mPitchAngle, 0.2f, 180.0f)`

Reply:
Good idea. Applied. Thanks